app/models/manageiq/providers/ansible_tower/shared/provider.rb

```diff
@@ -58,8 +58,69 @@ def url=(new_url)
     default_endpoint.url = uri.to_s
   end
 
+  def create_sandbox_data
+    with_provider_connection do |c|
+      organization = ensure_organization(c)
+      inventory = ensure_inventory(c, organization.id)
+      ensure_host(c, inventory.id)
+    end
+  end
+
+  def default_organization(connection = connect)
+    @@default_org ||= connection.api.organizations.all(
```

Comment:
Not sure why you are using a class variable.  In fact, given that those are inherited, I think that's generally bad right?